scan_subscriber_topic_name: /scan
scan_subscriber_queue_size: 10
proportional_param: 1

